TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/integ_test/integ_test/test_cases/multi_type_tests.py

```diff
@@ -91,9 +91,9 @@ def perform_operations_after_backfill_migration(self):
 
     def perform_operations_during_live_capture_migration(self):
         self.target_operations.get_document(cluster=self.target_cluster, index_name=self.index_name,
-                                            doc_id=self.doc_id3, max_attempts=10, delay=3.0)
+                                            doc_id=self.doc_id3, max_attempts=20, delay=3.0)
```

Comment:
Why are `max_attempts` and `delay` on every request, shouldn't these be in within common thresholds?

Reply:
This one is added AFTER backfill has finished, so it will take time for it to land, is that right?  Are those exponential backoffs? How many retries (and more importantly, lost seconds > optimal
) do you expect to happen, given the other parameters for this test?